templates/CRM/Contact/Form/Relationship.tpl

```diff
@@ -121,7 +121,7 @@
           </td>
         </tr>
         <tr class="crm-relationship-form-block-is_permission_b_a">
-          <td class="label"></td>
+          <td class="label"> </td>
```

Comment:
This just makes sure the two sets of permission "radios" are aligned properly on the form.  Otherwise the second set gets shifted across to the left where the label should be.